x/shield/keeper/migrations.go

```diff
@@ -6,6 +6,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	v232 "github.com/certikfoundation/shentu/v2/x/shield/legacy/v232"
+
+	v3 "github.com/certikfoundation/shentu/v2/x/shield/legacy/v3"
```

Comment:
it should be the old binary name, so it should be v240

Reply:
I viewed cosmos-sdk sources about migration, not sure I took it correctly. 
My understanding is that for a module upgrading, there could be two folders being created under x/***/legacy(or migrations)/. One, naming after the module's old version, is for holding the 'types' that would be used for reading from the store. The other one, naming after the module's new version, contains the code doing the migration logic. The new version of 'types' of the module could be just placed in its canonical folder, i.e. x/***/types/.
I think the current version of shield module running on main net is 2 and the new version could be 3. That's why I made v2 and v3 here.
Please correct me if I'm wrong or we should do it in our own way. @yoongbok-lee

Reply:
oh that makes sense. SGTM!